src/CSnakes.Runtime/Python/IPyBuffer.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace CSnakes.Runtime.Python;
+public interface IPyBuffer
+{
+    Int64 Length { get; }
+
+    bool Scalar { get; }
+
+    Span<Int32> AsInt32Scalar();
+    Span<Int64> AsInt64Scalar();
+    Span<UInt32> AsUInt32Scalar();
+    Span<UInt64> AsUInt64Scalar();
+    public Span<float> AsFloatScalar();
```

Comment:
```suggestion
    Span<int> AsInt32Scalar();
    Span<long> AsInt64Scalar();
    Span<uint> AsUInt32Scalar();
    Span<ulong> AsUInt64Scalar();
    public Span<float> AsFloatScalar();
```

Is there a reason `AsFloatScalar()` is marked `public`?

What about `AsDoubleScalar()`? Unsure if that is valid. I'm having trouble getting the valid values for the `format` field from the documentation.